### PR TITLE
Add GBC color shader and presets

### PR DESCRIPTION
**Pull request by Monroe88**
Adds a color correction shader for Game Boy Color. For now, it is directly based on Gambatte's color correction implementation.

### CODE REVIEW

File: handheld/shaders/color/gbc-color.cg [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+{
+     out_vertex OUT;
+#ifdef HLSL_4
+     float4 position = VIN.position;
+     float2 texCoord = VIN.texCoord;
+#else
+     OUT.Color = color;
+#endif
+     OUT.position = mul(modelViewProj, position);
+	OUT.texCoord = texCoord;
```
Comment (VelocityRa) [inline handheld/shaders/color/gbc-color.cg]: nit: there's a tab instead of spaces here

File: handheld/shaders/color/gbc-color.cg [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+                  
+    screen = mul(color, screen);
+    return screen;
+}
+
+float4 main_fragment(COMPAT_IN_FRAGMENT) : COMPAT_Output
+{
+    return gba_color(VOUT.texCoord);
+}
+COMPAT_END
```
Comment (VelocityRa) [inline handheld/shaders/color/gbc-color.cg]: lack of a newline at the end

File: handheld/lcd-shader-gbc-color.cgp [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+filter_linear1 = false
+filter_linear2 = false
+filter_linear3 = false
+filter_linear4 = false
+
+textures = BACKGROUND
+BACKGROUND = shaders/lcd-shader/background.png
+BACKGROUND_linear = true
+parameters = "display_gamma"
+display_gamma = "2.200000"
```
Comment (VelocityRa) [inline handheld/lcd-shader-gbc-color.cgp]: lack of a newline at the end

File: handheld/lcd-grid-v2-gbc-color.cgp [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+GSUBPIX_G = "0.750000"
+GSUBPIX_B = "0.000000"
+BSUBPIX_R = "0.000000"
+BSUBPIX_G = "0.000000"
+BSUBPIX_B = "0.750000"
+gain = "1.500000"
+gamma = "2.200000"
+blacklevel = "0.000000"
+ambient = "0.000000"
+BGR = "0.000000"
```
Comment (VelocityRa) [inline handheld/lcd-grid-v2-gbc-color.cgp]: lack of a newline at the end

**Comment by Monroe88** [comment]
I've fixed the style nits and newline issues. For some reason the comment for lcd-shader-gbc-color.cgp still shows despite the file being changed already.

**Comment by VelocityRa** [comment]
Cool, LGTM.

**Pull request merged**